eclair-core/src/main/scala/fr/acinq/eclair/wire/protocol/LightningMessageTypes.scala

```diff
@@ -263,7 +263,7 @@ case class Tor3(tor3: String, port: Int) extends OnionAddress { override def soc
 // @formatter:on
 
 case class NodeAnnouncement(signature: ByteVector64,
-                            features: Features,
+                            features: Features[NodeFeature],
```

Comment:
Fixed, I put untyped features inside `NodeAnnouncement` so it can deal with anything it receives but I restrict building `NodeAnnouncement` in our code to using node features.